src/components/seeds/database/index.tsx

```diff
@@ -325,6 +344,9 @@ export default function Database(props: DatabaseProps): JSX.Element {
       populateFieldOptions();
       populateSearchSummary();
     }
+    return () => {
+      activeRequests = false;
```

Comment:
this happens on unmount, when the state filters change and new requests are issued.